modules/distr-tracing-tempo-troubleshooting-issues-with-ingesting-traces.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="problems-ingesting-traces_{context}"]
+= Troubleshooting issues with ingesting traces
+
+When the TempoStack instance is failing to ingest traces, you can troubleshoot it as follows.
```

Comment:
Q: how would a user know if traces aren't being ingested? Would they see a particular error or run into particular issues that signal this is the case?

Reply:
@rubenvp8510, WDYT?

Reply:
There are different indicators, but the most important one is that the customer won't be able to see traces on the UI.

When that happens, is when should need to investigate if the traces are being droped, using the metrics mentioned in this doc.